pkg/scheduler/schedule_one.go

```diff
@@ -556,6 +569,9 @@ func (sched *Scheduler) findNodesThatPassFilters(
 	nodes []*framework.NodeInfo) ([]*framework.NodeInfo, error) {
 	numAllNodes := len(nodes)
 	numNodesToFind := sched.numFeasibleNodesToFind(fwk.PercentageOfNodesToScore(), int32(numAllNodes))
+	if len(sched.Extenders) == 0 && !sched.hasScoring(fwk) {
```

Comment:
This is not enough.

In an scenario where the scheduler only has a Bind extender, we could still set numNodesToFind to 1.

The check should be for `!hasScoring && !hasFilterExtenders`.

Reply:
Done